src/components/WaiveForm.tsx

```diff
@@ -69,6 +87,15 @@ const WaiveForm: React.FC<{}> = () => {
         setValidated(validated);
     };
 
+    const waiver_for = waive?.state ? getTestcaseName(waive.state) : '';
+    const {loading: qLoading, error: qError, data: metadata } =
+    useQuery<MetadataConsolidatedResult>(MetadataQuery, {
+        variables: { testcase_name: waiver_for },
+        errorPolicy: 'all',
+        fetchPolicy: 'network-only',
```

Comment:
This can be removed.
It is OK to use cache here.

Reply:
I am about entry: `fetchPolicy`

Reply:
Removed :+1: Thanks.